### PR TITLE
Fixed #24881: clarify order_with_respect_to documentation

### PR DESCRIPTION
**Pull request by karyon**
see https://code.djangoproject.com/ticket/24881.

### CODE REVIEW

**Comment by karyon** [comment]
i left the "usually" there and reworded the other paragraph like @alasdairnicol suggested. @timgraham, is there anything missing to merge this? (i didn't rebase yet to not lose the comments)

File: docs/ref/models/options.txt [hunk 1]
```diff
@@ -173,8 +173,8 @@ Django quotes column and table names behind the scenes.
 
 .. attribute:: Options.order_with_respect_to
 
-    Marks this object as "orderable" with respect to the given field. This is almost
-    always used with related objects to allow them to be ordered with respect to a
+    Makes this object orderable with respect to the given field, usually a 
+    ForeignKey. This can be used to make related objects orderable with respect to a
```
Comment (timgraham) [inline docs/ref/models/options.txt]: add double backticks around ForeignKey

File: docs/ref/models/options.txt [hunk 2]
```diff
@@ -218,12 +218,20 @@ Django quotes column and table names behind the scenes.
         >>> answer.get_previous_in_order()
         <Answer: 1>
 
+.. admonition:: order_with_respect_to implicitly sets the ordering option
+
+        Internally, ``order_with_respect_to`` adds an additional field/database
```
Comment (timgraham) [inline docs/ref/models/options.txt]: only 4 space indentation is needed for these admonitions

File: docs/ref/models/options.txt [hunk 3]
```diff
@@ -218,12 +218,20 @@ Django quotes column and table names behind the scenes.
         >>> answer.get_previous_in_order()
         <Answer: 1>
 
+.. admonition:: order_with_respect_to implicitly sets the ordering option
```
Comment (timgraham) [inline docs/ref/models/options.txt]: add double backticks around order_with_respect_to

File: docs/ref/models/options.txt [hunk 4]
```diff
@@ -218,12 +218,20 @@ Django quotes column and table names behind the scenes.
...
+
+        Internally, ``order_with_respect_to`` adds an additional field/database
+        column named ``_order``, and sets the model's :attr:`~Options.ordering`
```
Comment (timgraham) [inline docs/ref/models/options.txt]: chop comma

File: docs/ref/models/options.txt [hunk 5]
```diff
@@ -218,12 +218,20 @@ Django quotes column and table names behind the scenes.
...
+        option to this field. Consequently, ``order_with_respect_to`` and
+        ``ordering`` cannot be used together, and the ordering added by
+        ``order_with_respect_to`` will apply whenever you obtain a list of objects
+         of this model.
```
Comment (timgraham) [inline docs/ref/models/options.txt]: this line has an extra space of identation as the docs build failure complains.

**Comment by karyon** [comment]
@timgraham, revised according to your comments, squashed and rebased.

**Comment by timgraham** [comment]
merged in 27c839e0fce99254ad61322bb827a821f832e840, thanks! (By the way, you might want to configure your editor to remove trailing whitespace on save.)

**Pull request closed without merging**